pkgs/development/python-modules/undetected-chromedriver/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, selenium
+, requests
+, websockets
+}:
+
+buildPythonPackage rec {
+  pname = "undetected-chromedriver";
+  version = "3.5.4";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "ultrafunkamsterdam";
+    repo = pname;
```

```suggestion
    repo = "undetected-chromedriver";
```
it's preferable to just hardcode the reponame because some overrides might change the pname.